spdx-utils/build.gradle.kts

```diff
@@ -358,7 +358,10 @@ val generateLicenseRefTextResources by tasks.registering {
         licensesDir.walk().maxDepth(1).filter {
             it.isFile && it.extension == "yml" && !it.nameWithoutExtension.endsWith("-exception")
         }.forEach { file ->
-            val isSpdxLicense = file.readLines().any { it.startsWith("spdx_license_key: ") }
+            val isSpdxLicense = file.readLines().any {
+                it.startsWith("spdx_license_key: ") && !it.substringAfter("spdx_license_key: ").trim()
+                    .startsWith("LicenseRef-")
+            }
```

Comment:
According to our (now written down 😀 ) rules, we'd need a bank line after this one.

Reply:
Done.